src/runtimes/runtime.ts

```diff
@@ -39,14 +39,15 @@ export interface ZipFunctionResult {
   bundlerErrors?: object[]
   bundlerWarnings?: object[]
   config: FunctionConfig
+  displayName?: string
+  generator?: string
   inputs?: string[]
   includedFiles?: string[]
   inSourceConfig?: ISCValues
   nativeNodeModules?: object
   nodeModulesWithDynamicImports?: string[]
   path: string
-  displayName?: string
-  generator?: string
+  runtimeName?: string
```

Comment:
My understanding is that this now contains the version (e.g. `nodejs18.x`), so it's technically not just a runtime _name_ but also version?

Reply:
Any idea what I should name it?